### ISSUE TITLE
Ability to add unsaved models to saved models.

### ISSUE THREAD

**Issue by allynt** [ES-DOC/esdoc-questionnaire#481]
WHO: @allynt

Ability to add unsaved models to saved models.

**Comment by allynt** [comment]
I can confirm this is not quite working yet.

**Comment by allynt** [comment]
Figured out what is going on...

When I serialize a model to JSON, I first work out if the model is unsaved or saved.  If it is unsaved, I assume that I used my cleverness to overcome the default Django limitations regarding relationship fields to unsaved models.  If it is saved, I assume that I should just do the standard Django things.  The problem is that that latter approach results in me missing out the newly-added-but-as-yet-unsaved things.  

The solution is to always combine saved and unsaved relationship fields.
